main.go

```diff
@@ -226,8 +229,8 @@ func main() {
 			*port = "8001"
 		}
 		log.Infof("listen on port: %s", *port)
-		log.Fatal(r.Run(":" + *port))
+		log.Error(r.Run(":" + *port))
 	default:
-		log.Fatalf("unsupported mode %s", *mode)
```

Comment:
Let's keep them as `Fatal` to get an exit error that is not 0

Reply:
Gonna move to `Panicf`. As we discussed Fatal triggers `os.Exit` which doesn't run defers.